tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -1580,6 +1580,26 @@ public TezConfiguration(boolean loadDefaults) {
       TEZ_PREFIX + "generate.debug.artifacts";
   public static final boolean TEZ_GENERATE_DEBUG_ARTIFACTS_DEFAULT = false;
 
+  /**
+   * Int value. Property size threshold (in bytes) for logging during payload serialization. Properties exceeding the
+   * threshold are considered unusually large and potentially problematic thus they should be logged.
+   */
+  @ConfigurationScope(Scope.VERTEX)
+  @ConfigurationProperty(type="integer")
+  public static final String TEZ_LOGGING_PROPERTY_SIZE_THRESHOLD =
+      TEZ_PREFIX + "logging.property.size.threshold";
+  public static final int TEZ_LOGGING_PROPERTY_SIZE_THRESHOLD_DEFAULT = 512 * 1024;
+  /**
```

Comment:
nit, we tend to add a line between every config properties, even if they are related to each other

Reply:
Fixed in https://github.com/apache/tez/pull/308/commits/d8215af3d6789b6dfad88676dfd0cc19c6612d34